### PR TITLE
UX changes based on user feedback from the card design changes

### LINKED ISSUE #370: [UX] Title should be on the top of the card
Number of requests (2)

### LINKED ISSUE #371: [UX] Title font should be bigger

### LINKED ISSUE #372: [UX] Hide comments when 0 comments
Please open new issues for HuBoard [here](https://github.com/huboard/huboard/issues/new)
_contact_ support@huboard.com _for personal support on critical issues_

### PR DESCRIPTION
**Pull request by rauhryan**
closes #370 
closes #371 
closes #372

### CODE REVIEW

**Comment by discorick** [comment]
What do you think of the PR icon beside the title? 

image

File: ember-app/app/models/new/issue.js [hunk 1]
```diff
@@ -244,6 +244,7 @@ var Issue = Model.extend(IssueFiltersMixin, Messaging, CardSubscriptions, {
       correlationId: this.get("correlationId")
     }).then(function(response){
         this.set("processing", false);
+        this.incrementProperty('data.comments');
```
Comment (discorick) [inline ember-app/app/models/new/issue.js]: Good catch!

Comment (rauhryan) [inline ember-app/app/models/new/issue.js]: 👍

**Comment by rauhryan** [comment]
> What do you think of the PR icon beside the title?

I like the idea, but we should move that to a new issue

**Comment by rauhryan** [comment]
Opened #375 for further discussion of moving the type icons around

**Pull request merged**